test/xcribe/document_test.exs

```diff
@@ -89,7 +89,7 @@ defmodule Xcribe.DocumentTest do
     end
 
     test "dont document when env var is not defined", %{conn: conn} do
-      Application.delete_env(:xcribe, :env_var)
+      Application.put_env(:xcribe, :env_var, "NOT_DEFINED_ENV_VAR_TEST")
```

Comment:
@Danielwsx64 is there a reason for not using `Application.delete_env`?

Reply:
When `env_var` config is not defined `Xcribe` looks for `XCRIBE_ENV` environment. If for some reason this env is defined with some value this test fail. So is better define some random enviroment to avoid false negative.